lib/matplotlib/dates.py

```diff
@@ -1944,8 +1944,10 @@ def register_converters(cls):
         """
         if cls.conv_st == 'concise':
             converter = ConciseDateConverter
-        else:
+        elif cls.conv_st == 'auto':
             converter = DateConverter
+        else:
+            return
```

Comment:
Is silently returning (instead of raising a warning or error) the right thing to do here?

Reply:
I'll catch where its set...